app/views/govuk_publishing_components/components/_input.html.erb

```diff
@@ -107,6 +113,7 @@
     autofocus: autofocus,
     class: css_classes,
     data: data,
+    dir: right_to_left_field ? "rtl" : nil,
```

Comment:
I might not be fully understanding this, so please feel free to correct me - there are three rtl options: the normal one, `field` and `help`. `help` is applied to the label, hint and error message, and `field` is applied to the input itself. But the normal one isn't applied directly to any elements, it's just used to determine the others, is that right?

If that's correct, could `right_to_left` not be directly applied to the input, and `field` removed? 

```Ruby
right_to_left ||= false
right_to_left_help = right_to_left_help.nil? ? right_to_left : right_to_left_help

# label: right_to_left_help
# hint: right_to_left_help
# error: right_to_left_help
# input: right_to_left ? "rtl" : nil
```

Reply:
Hmm, yes, that makes sense. And will apply equally to the textarea component. I've updated the input and textarea components to reflect this with changes to the example documentation, the html and the test.